ports/onnxruntime/portfile.cmake

```
@@ -0,0 +1,348 @@
+    WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${ONNXRUNTIME}
+    LOGNAME build-${TARGET_TRIPLET})
+
+  #
+  vcpkg_execute_required_process(
+    COMMAND ${GIT} submodule sync --recursive
+    WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${ONNXRUNTIME}
+    LOGNAME build-${TARGET_TRIPLET})
+
+  #
+  vcpkg_execute_required_process(
+    COMMAND ${GIT} submodule update --init --recursive
+    WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${ONNXRUNTIME}
+    LOGNAME build-${TARGET_TRIPLET})
+
+else()
+  vcpkg_execute_required_process(
+    COMMAND ${GIT} clone --recursive ${ONNX_GITHUB_URL} ${ONNXRUNTIME}
+    WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}
+    LOGNAME build-${TARGET_TRIPLET})
+endif()
+
+
+set(SOURCE_PATH "${CURRENT_BUILDTREES_DIR}/${ONNXRUNTIME}")
+
+vcpkg_configure_cmake(
+  SOURCE_PATH "${SOURCE_PATH}/cmake"
+  PREFER_NINJA
+  OPTIONS
+    -Donnxruntime_RUN_ONNX_TESTS=OFF
+    -Donnxruntime_BUILD_WINML_TESTS=ON
+    -Donnxruntime_GENERATE_TEST_REPORTS=ON
+    -Donnxruntime_USE_CUDA=OFF
+    -Donnxruntime_USE_FEATURIZERS=OFF
+    -Donnxruntime_USE_JEMALLOC=OFF
+    -Donnxruntime_USE_MIMALLOC_STL_ALLOCATOR=OFF
+    -Donnxruntime_USE_MIMALLOC_ARENA_ALLOCATOR=OFF
+    -Donnxruntime_BUILD_SHARED_LIB=OFF
+    -Donnxruntime_USE_EIGEN_FOR_BLAS=ON
+    -Donnxruntime_USE_OPENBLAS=OFF
+    -Donnxruntime_USE_DNNL=OFF
+    -Donnxruntime_USE_MKLML=OFF
+    -Donnxruntime_USE_NNAPI_BUILTIN=OFF
+    -Donnxruntime_USE_RKNPU=OFF
+    -Donnxruntime_USE_OPENMP=OFF
+    -Donnxruntime_USE_TVM=OFF
+    -Donnxruntime_USE_LLVM=OFF
+    -Donnxruntime_ENABLE_MICROSOFT_INTERNAL=OFF
+    -Donnxruntime_USE_VITISAI=OFF
+    -Donnxruntime_USE_NUPHAR=OFF
+    -Donnxruntime_USE_TENSORRT=OFF
+    -Donnxruntime_TENSORRT_HOME=
+    -Donnxruntime_USE_MIGRAPHX=OFF
+    -Donnxruntime_MIGRAPHX_HOME=
+    -Donnxruntime_CROSS_COMPILING=OFF
+    -Donnxruntime_DISABLE_CONTRIB_OPS=OFF
+    -Donnxruntime_DISABLE_ML_OPS=OFF
+    -Donnxruntime_DISABLE_RTTI=OFF
+    -Donnxruntime_DISABLE_EXCEPTIONS=OFF
+    -Donnxruntime_DISABLE_ORT_FORMAT_LOAD=OFF
+    -Donnxruntime_MINIMAL_BUILD=OFF
+    -Donnxruntime_ENABLE_LANGUAGE_INTEROP_OPS=OFF
+    -Donnxruntime_USE_DML=OFF
+    -Donnxruntime_USE_WINML=OFF
+    -Donnxruntime_USE_TELEMETRY=OFF
+    -Donnxruntime_ENABLE_LTO=OFF
+    -Donnxruntime_USE_ACL=OFF
+    -Donnxruntime_USE_ACL_1902=OFF
+    -Donnxruntime_USE_ACL_1905=OFF
+    -Donnxruntime_USE_ACL_1908=OFF
+    -Donnxruntime_USE_ACL_2002=OFF
+    -Donnxruntime_USE_ARMNN=OFF
+    -Donnxruntime_ARMNN_RELU_USE_CPU=ON
+    -Donnxruntime_ARMNN_BN_USE_CPU=ON
+    -Donnxruntime_ENABLE_NVTX_PROFILE=OFF
+    -Donnxruntime_ENABLE_TRAINING=ON
+    -Donnxruntime_USE_HOROVOD=OFF
+    -Donnxruntime_USE_NCCL=ON
+    -Donnxruntime_BUILD_BENCHMARKS=OFF
+    -Donnxruntime_USE_ROCM=OFF
+    -Donnxruntime_PYBIND_EXPORT_OPSCHEMA=OFF
+    -Donnxruntime_ENABLE_MEMLEAK_CHECKER=OFF
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    -DONNX_ML=ON
+    -DONNX_NAMESPACE=onnx
+    ${FEATURE_OPTIONS}
```

```suggestion
    ${FEATURE_OPTIONS}
    -DPYTHON_EXECUTABLE=${PYTHON3}
```
You may try to find python3 in this way.

---

done. Thanks.